src/makerspace/templates/makerspace/statistics.html

```diff
@@ -0,0 +1,68 @@
+{% extends 'web/base.html' %}
+{% load static %}
+{% load i18n %}
+
+
+{% block title %}{% translate "Statistics" %}{% endblock title %}
+
+{% block extra_head %}
+    <script defer src="{% static 'makerspace/js/statistics_charts.js' %}"></script>
+{% endblock extra_head %}
+
+{% block body%}
+<div class="ui container">
+    <h1 class="ui text-centered make-col-blue header"> {% translate "Statistics" %}</h1>
+    <div class="ui column cards">
+    <div class="card " style="height: 50vh; width: 65vh">
+            <div class="content">
+            <div class="header">{% translate "How long each 3D-printer* has been reserved in total:" %} </div>
```

Comment:
Maybe the plots would look cleaner if instead of showing the asterisk text at the bottom of the plots, the word "3D-printer*" was instead put in a <span>-element, and the asterisk text included in the title attribute. Then, the text would only be shown if someone is hovering over the word "3D-printer*". To make it visible that this word was different, the span element could be styled differently than the rest of the text (not necessarily the same way as I did it in the example, because that looks ugly), and also given a pointer cursor. But this is only a suggestion – not something that absolutely needs to be changed.
```suggestion
            o	<div class="header"> {% blocktranslate %}Number of times each <span title="Inkluderer 3D-printere, Raise3D-printere og SLA 3D-printere" style="background: yellow; cursor: pointer;">3D-printer*</span> has been reserved:{% endblocktranslate %} </div>
```